presto-main/src/main/java/io/prestosql/execution/DynamicFilterConfig.java

```diff
@@ -35,6 +35,7 @@
 {
     private boolean enableDynamicFiltering = true;
     private boolean enableLargeDynamicFilters;
+    private int serviceThreadCount = 2;
```

Comment:
previously it was 1, inentional?

Reply:
I think this threads might now have more work when `enable_large_dynamic_filters` is enabled